protos/feast/core/DataSource.proto

```diff
@@ -89,10 +89,18 @@ message DataSource {
     // s3://path/to/file for AWS S3 storage
     // gs://path/to/file for GCP GCS storage
     // file:///path/to/file for local storage
+    // Deprecated in Feast 0.23.0
     string file_url  = 2;
```

Comment:
You should change this in place; serialized protobufs only use the index number not the field name.

Reply:
Ah I didn't know that but that makes a lot of sense! Thanks for the suggestion that makes the update so much easier.